gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -132,6 +132,8 @@ private static void configureErrorProneOptions(
                 "CatchSpecificity",
                 "CanIgnoreReturnValueSuggester",
                 "InlineMeSuggester",
+                // We often use javadoc comments without javadoc parameter information.
+                "NotJavadoc",
```

Comment:
It feels wrong to disable this. We should not be using Javadoc comments (`/**`) in places where Javadoc is not supported. In those places we should using regular multi-line comments (`/*`).